src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+$first-color: #060b35;
+$second-color: #616070;
+$button-color: #00acdc;
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
you don't have to set width here, let the content define total width

Reply:
> you don't have to set width here, let the content define total width

Could you help me? If I disable this property, everything looks broken.